### ISSUE TITLE
PDO Error using PostgreSQL when visiting Dashboard - Part 2

### ISSUE THREAD

**Issue by Timusan** [verbb/formie#1244]
### Describe the bug

Hi Josh

This [old issue](https://github.com/verbb/formie/issues/222) seems to be back. Since the recent upgrade (Formie 1.6.21, Craft 3.7.63.1):

```
Undefined function: 7 ERROR:  operator does not exist: integer ~~ unknown
LINE 6: WHERE ("formie_submissions"."formId" LIKE $1) AND ("formie_s...
```

This is a PostgreSQL install. Could this be regression?

### Steps to reproduce

1. Install Formie on a Craft installation running on PostgreSQL 13
2. Visit the forms overview dashboard

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

3.7.63.1

### Plugin version

1.6.21

**Comment by Timusan** [comment]
One important difference, this is not on the form overview page, but on the dashboard when adding the Formie widget.

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.21",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.6.22](https://github.com/verbb/formie/releases/tag/1.6.22)
